app-directory/redirect-with-fallback/app/page.tsx

```diff
@@ -0,0 +1,23 @@
+import { Page, Text, Link, Code, Snippet } from '@vercel/examples-ui'
+
+export default function Home() {
+  return (
+    <Page className="flex flex-col gap-12">
+      <section className="flex flex-col gap-6">
+        <Text variant="h1">Redirects</Text>
+        <ul>
+          <li>
+            <Link href="/simple-redirect-source">/simple-redirect-source</Link>{' '}
+            - handled in API Route
```

Comment:
I think this should be "Route Handler". Also make it an external `<Link>` to https://nextjs.org/docs/app/building-your-application/routing/router-handlers